app/models/pawn.rb

```diff
@@ -5,23 +5,42 @@ def get_image
   end
 
   def valid_move?(x,y)
-    super(x,y) && (valid_one_forward_move?(x,y) || valid_two_forward_move?(x,y) || valid_standard_capture?(x,y))
+    return false unless super(x,y)
+    return false unless valid_white_move?(x,y) || valid_black_move?(x,y)
+    valid_single_move?(x,y) || valid_first_move?(x,y) || valid_standard_capture?(x,y)
   end
 
-  def valid_one_forward_move?(x,y)
-    square_is_vacant?(x,y) && (x == x_coord) && (y == y_coord + self.movement_direction)
+  def valid_single_move?(x,y)
+    x_values(x) == 0 && 
+    y_values(y) == 1
   end
 
-  def valid_two_forward_move?(x,y)
-    square_is_vacant?(x,y) && (x == x_coord) && (y == y_coord + (self.movement_direction * 2)) && has_not_moved?
+  def valid_first_move?(x,y)
+    move_count == 0 && 
+    x_values(x) == 0 && 
+    y_values(y) <= 2
   end
 
   def valid_standard_capture?(x,y)
-    (x - x_coord).abs == 1 && (y == y_coord + self.movement_direction) && opponent_on_square?(x,y)
+    opponent_on_square?(x,y) &&
+    x_values(x) == 1 && 
+    y_values(y) == 1  
   end
 
-  def movement_direction
-    white? ? 1 : -1
+  def valid_white_move?(x,y)
+    self.white? && (y - y_coord) > 0
+  end
+
+  def valid_black_move?(x,y)
+    !self.white? && (y - y_coord) < 0
+  end
+
+  def en_passant_capturable?(x,y)
```

Comment:
should the name be rather `can_capture_en_passant?` - meaning from the viewpoint of the pawn we are looking at.. capturable sound a bit as we were checking if this pawn can be captured.. hope it makes sense..